modules/post-installation-adding-nutanix-failure-domains-control-planes.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-control-planes_{context}"]
+= Distributing control planes across failure domains
+
+You distribute control planes across Nutanix failure domains by modifying the control plane machine set custom resource (CR).
+
+.Prerequisites
+
+* You have added the failure domains to the cluster's Infrastructure custom resource definition (CRD).
+* The cluster has an activated and functioning Control Plane Machine Set Operator.
+
```

Comment:
Should also add this as a Prerequisite:
* The control plane machine set custom resource (CR) is in active state (the CR's spec.state is "Active").

Reply:
In speaking with @jeana-redhat what we had was effectively saying the same thing. However, I went with your language. I also added a cross reference to the docs on how to check on the state of the control plane machine set CR.